mlir/test/Target/LLVMIR/Import/import-failure.ll

```diff
@@ -12,15 +12,6 @@ bb2:
 
 ; // -----
 
-; CHECK:      <unknown>
-; CHECK-SAME: error: unhandled value: ptr asm "bswap $0", "=r,r"
-define i32 @unhandled_value(i32 %arg1) {
-  %1 = call i32 asm "bswap $0", "=r,r"(i32 %arg1)
```

Comment:
Ideally we could keep a test that runs into an unhandled value error. Are you by chance aware of an alternative test case? All cases I can think of run into another error first (e.g. in the unhandled instruction error above).

Reply:
not that I could think of. As an example, tried the callbr version (since I only added handling for call), but that first hit a bad conversion for the basicblock which was considered a constant not value

Reply:
Ok let's drop the test then.